### PR TITLE
Adds a vacation spots element to the home page

### PR DESCRIPTION
**Pull request by cjoshea9**
Adds an element at the bottom of the home screen where the comments section will end up going. For now takes a list of vacation destinations from server side and inputs them as li elements.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -42,8 +42,24 @@ function showSlide(n) {
...
+  vacationList.appendChild(first);
+
+  let second = document.createElement("li");
+  second.innerText = places.second;
+  vacationList.appendChild(second);
+
+  let third = document.createElement("li");
+  third.innerText = places.third;
+  vacationList.appendChild(third);
+
```
Comment (Danish21) [inline portfolio/src/main/webapp/script.js:63]: Uber Nit: remove extra new lines

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -42,8 +42,24 @@ function showSlide(n) {
   slides[slideIndex].style.display = "block";
 }
 
-async function getInitialGreeting() {
+async function getVacationSpots() {
   const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById('data-servlet').innerText = greeting;
+  const places = await response.json();
+  console.log(places);
```
Comment (Danish21) [inline portfolio/src/main/webapp/script.js:48]: Nit:
Console.log should only be used for testing purposes and removed before submission.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -15,18 +15,31 @@
...
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Cory! This should be rendered on the home page now.");
+
+    Map<String, String> vacations = new HashMap<>();
+    vacations.put("first", "Australia");
```
Comment (Danish21) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:34]: If the keys to the HashMap are essential numeric i.e. first, second, etc, then I would suggest using Array/ArrayList as you can simply append to it without having to create string versions of numeric keys.

Comment (cjoshea9) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:34]: Okay. I was trying to play around with using a HashMap versus and ArrayList, but for this context you are right that an ArrayList would be better.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -42,8 +42,14 @@ function showSlide(n) {
...
 }
 
-async function getInitialGreeting() {
+async function getVacationSpots() {
   const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById('data-servlet').innerText = greeting;
+  const places = await response.json();
+  const vacationList = document.getElementById('data-servlet');
+  vacationList.innerHTML = "<h1>Vacation Spots</h1>";
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:49]: I encourage you to use `createElement` here too. It'll make it much more difficult to accidentally open yourself up to XSS.

**Comment by Danish21** [comment]
LGTM

**Pull request merged**